pkg/syncer/util.go

```diff
@@ -92,6 +92,29 @@ func getBoundPVs(ctx context.Context, metadataSyncer *metadataSyncInformer) ([]*
 	return boundPVs, nil
 }
 
+// getPVCsInPendingState is a helper function for fetching PVCs not in Bound state.
+func getPVCsInPendingState(ctx context.Context, metadataSyncer *metadataSyncInformer) ([]*v1.PersistentVolumeClaim,
+	error) {
+	log := logger.GetLogger(ctx)
+	var pendingPVCs []*v1.PersistentVolumeClaim
+	// Get all PVCs from kubernetes.
+	allPVCs, err := metadataSyncer.pvcLister.List(labels.Everything())
+	if err != nil {
```

Comment:
Do we get a non-nil error if there are no PVCs?

Reply:
We get a nil for zero PVCs